spec/support/webmock.rb

```diff
@@ -4,16 +4,25 @@
     opts = args.last.is_a?(Hash) ? args.pop : {}
     ver = args.first || opts[:api_version] || 1
     fmt = opts[:api_format] || :json
-    a_request(method, Gemfury.endpoint + path).with(
+    with_opts = {
       :headers => {
-        'Accept' => "application/vnd.fury.v#{ver}+#{fmt}",
+        'Accept' => 'application/vnd.fury.v%s+%s' % [ ver, fmt ],
```

Comment:
Curious why you prefer `'application/vnd.fury.v%s+%s' % [ ver, fmt ]` over `"application/vnd.fury.v#{ver}+#{fmt}"` here?

Reply:
It's an old convention i have, copied from the time i code in C. I prefer the `sprintf` way of composing strings and variables, instead of the insertion style, which i first encountered in PHP. I find it much easier to read the string literal, easy to replace with a constant later. Though i haven't confirmed, there might be compiler benefits, given the string can be stored in a static way, instead of being dynamic and interpreted in real-time.